src/find.js

```diff
@@ -1,6 +1,6 @@
 import Quad from "./quad.js";
 
-export default function(x, y, radius) {
+export default function(x, y, radius, filter) {
```

Comment:
how about defaulting filter to false like
`export default function(x, y, radius, filter = false)`
so old code won't break?